en/advanced_config/filter_tuning.md

```diff
@@ -0,0 +1,80 @@
+# Filter Tuning & Control Latency
+
+Filters can be used to trade off [control latency](#control-latency), which affects flight performance, and noise filtering, which impacts both flight performance and motor health.
+
+This topic provides an overview of control latency and PX4 filter tuning.
+
+## Control Latency
+
+The *control latency* is the delay from a physical disturbance of the vehicle until the motors react to the change.
+
+:::tip
+Lowering latency allows you to increase the rate **P** gains, which results in better flight performance.
+Even one millisecond difference in the latency can have a significant impact.
+:::
+
+The following factors affect control latency:
+- A soft airframe or soft vibration mounting increases latency (they act as a filter).
+- Low-pass filters in software and on the sensor chip trade off increased latency for improved noise filtering.
+- PX4 software internals: the sensor signals need to be read in the driver and then pass through the controller to the output driver.
+- The IO chip (MAIN pins) adds about 5.4 ms latency compared to using the AUX pins (this does not apply to a *Pixracer* or *Omnibus F4*, but does apply to a Pixhawk).
+  To avoid the IO delay, disable [SYS_USE_IO](../advanced_config/parameter_reference.md#SYS_USE_IO) and attach the motors to the AUX pins instead.
+- PWM output signal: enable One-Shot to reduce latency ([PWM_RATE](../advanced_config/parameter_reference.md#PWM_RATE)=0).
+
+Below we look at the impact of the low pass filters.
+
+## Filters
+
+This is the filtering pipeline for the controllers in PX4:
+- On-chip DLPF for the gyro sensor.
+  The cutoff frequency is set to 98Hz and it is sampled at 1kHz.
```

Comment:
This one is now disabled on all chips where it can be disabled.

Reply:
We should mention that the sampling and filtering is also performed at the full raw sensor rate. https://github.com/PX4/PX4-Autopilot/blob/58b40fbbb921ad306f97da040acd7f141496484c/src/modules/sensors/vehicle_angular_velocity/imu_gyro_parameters.c#L85-L105

Reply:
I really don't mind if you make the changes directly rather than telling me. Reduces changes of me misunderstanding.

Anyway, I have updated this, please check. @dagar in particular, I have put the comment on sampling after the line below starting `It can be configured with the [IMU_GYRO_CUTOFF]` because I think you're only talking about the IMU_GYRO_CUTOFF